justfile

```diff
@@ -61,4 +61,50 @@ make-tn-models project_url api_key endpoint='/api/users/':
     echo "Please run 'tn install-bun' to install bunx first."
     exit 1
   fi
-  bunx @thinknimble/tnm-cli read '{{project_url}}/api/schema/?format=yaml' -t '{{api_key}}' -u '{{endpoint}}'
\ No newline at end of file
+  bunx @thinknimble/tnm-cli read '{{project_url}}/api/schema/?format=yaml' -t '{{api_key}}' -u '{{endpoint}}'
+
+#
+# GitHub CLI
+#
+[group('github')]
+gh-install:
+  #!/usr/bin/env bash
+  if ! command -v gh &> /dev/null; then
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+      brew install gh
+    elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
+      type -p curl >/dev/null || sudo apt install curl -y
+      curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+      && sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
+      && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+      && sudo apt update \
+      && sudo apt install gh -y
```

Comment:
Ugh, Ubuntu install steps are ugly.

Reply:
I like the idea of having the entire script in view in the primary `justfile`, but it may be a better standard to move each of these out to a library of scripts and have the `justfile` simply wrap them.